diff --git a/typescript-jest/src/filter.ts b/typescript-jest/src/filter.ts
index fe10215..ab68aac 100644
--- a/typescript-jest/src/filter.ts
+++ b/typescript-jest/src/filter.ts
@@ -1,5 +1,5 @@
 export type Item = { name: string; category: string };
 
 export function byCategory(items: Item[], category: string): Item[] {
-  return items.filter((item) => item.category === category);
+  return items.slice(0, 0);// return items.filter((item) => item.category === category);
 }
